CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 
 - `sandstone/Picker` value to not marquee when changing `title`
 - `sandstone/Scroller` and `sandstone/VirtualList` to scroll by hover when scrollbar is hidden
+- `sandstone/Scroller` and `sandstone/VirtualList` to focus elements on scroll edges when `hoverToScroll` is `true`
```

Comment:
```suggestion
- `sandstone/Scroller` and `sandstone/VirtualList` to focus elements on scroll start and end positions  when `hoverToScroll` is `true`
```

Reply:
I updated from 'on scroll edges' to 'at scroll boundaries'.